factor/totp/db/install.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" ?>
-<XMLDB PATH="admin/tool/mfa/factor/totp/db" VERSION="20191104" COMMENT="XMLDB file for Moodle tool/mfa plugin"
+<XMLDB PATH="admin/tool/mfa/factor/totp/db" VERSION="20191104" COMMENT="XMLDB file for Moodle tool/mfa plugin - totp factor"
```

Comment:
rename this table from tool_mfa_factor_totp to factor_totp

Reply:
Corrected